metadata/demographic/census_blocks_2020.md

```diff
@@ -0,0 +1,90 @@
+Census Blocks are the smallest geographic unit used by the United States Census Bureau for tabulating population information. Due to privacy concerns, only a limited amount of data is available at the block level.
+
+## Description
+
+<!-- A more in-depth explanation of the dataset, where it came from and how its created, so the user can decide if its what they need. -->
+
+### What is the dataset
+
+Census blocks are small geographic zones through which population data can be collected and organized. These blocks are often bounded by physical features such as roads or rivers, as well as municipal boundaries such as city limits or property lines. This dataset includes the census blocks for the State of Utah, current as of the 2020 Census and represented as polygons.
+
+### What is the purpose of the dataset
+
+Geography is a fundamental aspect of the Census, providing the framework for the once-a-decade count of population and housing. Read about some of the key geography changes in the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).
+
+### What does the dataset represent
+
+Each Census block in this dataset is given a unique four-digit Census Block Number. For more information on how to interpret the Census Block Number, refer to this article from the [US Census Bureau](https://www.census.gov/newsroom/blogs/random-samplings/2011/07/what-are-census-blocks.html). While census blocks include information on major racial groups, housing, and population, it is important to note that census blocks are delineated by physical and municipal boundaries, not by population size. Thus, a larger block does not denote a larger population size. In fact, many census blocks contain a population number of zero (such as in the case of water-only areas).
+
+### How was the dataset created
+
+Utah's 2020 census geographic boundaries became available in February 2021 and the demographic redistricting legacy files (the 2020 Census Redistricting P.L. 94-171 Summary Files) became available on August 12, 2021. The Census Bureau will release a more user-friendly version of the 2020 Census Redistricting P.L. 94-171 Summary Files on September 30, 2021. In the meantime, UGRC, with collaboration from the Analytics Group at [WFRC](https://wfrc.org/), has worked with the August 12 release and has created ready-to-use GIS data in several formats that improve the user-friendliness of the 2020 redistricting data. Read more about these offerings in the blog post, 2020 Census Redistricting Data in Easy to Use GIS Formats
```

Comment:
That paragraph is from the original data page, I kept it in because I wasn't sure if we were wanting to use it for the new metadata or not. Now that we've decided to keep the info in, I'll go ahead and change the tenses/verbage so it makes sense.